### ISSUE TITLE
NAWP is not available in the released dataset

### ISSUE THREAD

**Issue by Coronal-Halo** [dasongli1/SnapUGC_Engagement#3]
Hi,

First, thank you for making the code and dataset public.

However, in the .csv files released, there is no NAWP score of any video, but only ECR for some videos in the training dataset. 

Is this a mistake? How can we get the ECR metric too?

Thank you,

**Comment by dasongli1** [comment]
Hi, we are holding a ICCV competition of https://codalab.lisn.upsaclay.fr/competitions/23005. 

Please note that the NAWP is temporarily hidden to ensure fair competition. It will be released after the competition concludes. 

Thank you for your understanding.

**Comment by Imane219** [comment]
> Hi, we are holding a ICCV competition of https://codalab.lisn.upsaclay.fr/competitions/23005.
> 
> Please note that the NAWP is temporarily hidden to ensure fair competition. It will be released after the competition concludes.
> 
> Thank you for your understanding.

Hi, thanks again for all your hard work on the dataset and the competition!

Now that the competition has wrapped up, as per your earlier announcement, would it now be possible to release the ECR results for the validation set and the final NAWP scores?

This would be a great help for our team to complete our analysis. Thanks in advance for your help!

**Comment by MarcosTG1** [comment]
Hi,

Following up on Imane219's message from September 2025.

I'm a final-year Data Science student at Universitat de València 
(Spain) currently reproducing the SnapUGC pipeline for my BSc thesis 
under academic supervision.

Now that the ICCV VQualA 2025 challenge has concluded, would it be 
possible to release:

(a) ECR labels for val_data.csv, and  
(b) NAWP scores for the training set

as originally announced?

This would directly unblock my thesis experimental chapter. I would 
of course cite your work prominently and comply with any data usage 
conditions.

Thank you for the incredible resource you've made available to the 
community.

**Comment by githubier** [comment]
> Hi,
> 
> Following up on Imane219's message from September 2025.
> 
> I'm a final-year Data Science student at Universitat de València (Spain) currently reproducing the SnapUGC pipeline for my BSc thesis under academic supervision.
> 
> Now that the ICCV VQualA 2025 challenge has concluded, would it be possible to release:
> 
> (a) ECR labels for val_data.csv, and (b) NAWP scores for the training set
> 
> as originally announced?
> 
> This would directly unblock my thesis experimental chapter. I would of course cite your work prominently and comply with any data usage conditions.
> 
> Thank you for the incredible resource you've made available to the community.

Hi ,
Saw your old message about requesting SnapUGC data for your thesis. I also need val & test data for our VQA research.
Just wondering, have you received the dataset from authors? If yes, would you mind sharing it with me? 
Thanks!

**Comment by githubier** [comment]
> > Hi, we are holding a ICCV competition of https://codalab.lisn.upsaclay.fr/competitions/23005.
> > Please note that the NAWP is temporarily hidden to ensure fair competition. It will be released after the competition concludes.
> > Thank you for your understanding.
> 
> Hi, thanks again for all your hard work on the dataset and the competition!
> 
> Now that the competition has wrapped up, as per your earlier announcement, would it now be possible to release the ECR results for the validation set and the final NAWP scores?
> 
> This would be a great help for our team to complete our analysis. Thanks in advance for your help!

hi, have you received the dataset from authors? If yes, would you mind sharing it with me?
Thanks!
